Source/WebKit/NetworkProcess/Classifier/ResourceLoadStatisticsStore.cpp

```diff
@@ -2173,7 +2194,12 @@ bool ResourceLoadStatisticsStore::predicateValueForDomain(SQLiteStatementAutoRes
         ITP_RELEASE_LOG_ERROR(m_sessionID, "%p - ResourceLoadStatisticsStore::predicateValueForDomain failed to bind, error message: %" PRIVATE_LOG_STRING, this, m_database.lastErrorMsg());
         return false;
     }
-    return !!predicateStatement->columnInt(0);
+    return predicateStatement->columnInt(0);
+}
+
+bool ResourceLoadStatisticsStore::predicateBoolValueForDomain(SQLiteStatementAutoResetScope& predicateStatement, const RegistrableDomain& domain) const
+{
+    return !!predicateValueForDomain(predicateStatement, domain);
```

Comment:
Might be better to let caller get int value and convert to bool by themselves if needed.